strapi/strapi
# [v5] Replace usage of http/https for requests with fetch api

### What does it do?

Replace all http/https requests with fetch api requests

- currently only one place in data-transfer but if I find more I'll add them here

### Why is it needed?

We shouldn't be using http/https directly, and it in fact led to [a bug](https://github.com/strapi/strapi/issues/17481) in the past

Also, trying to standardize everything to use fetch api

### How to test it?

Affects export and transfer --from, so both should be tested to ensure they still work

It should work with http and https servers

### Related issue(s)/PR(s)

Prevents issues like this https://github.com/strapi/strapi/issues/17481 from happening again
